iroha/src/smartcontracts/isi/mod.rs

```diff
@@ -107,6 +107,16 @@ pub enum MathError {
     OverflowError,
 }
 
+/// Block with parent hash not found struct
+#[derive(Debug, Clone, Copy)]
+pub struct ParentHashNotFound(pub Hash);
```

Comment:
Also could you implement `std::error::Error` for it

Reply:
Done.